content/blog/2016/2016-05-18-external-workspace-manager-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "External Workspace Manager Plugin"
```

Comment:
IMHO needs something different, because it looks like announcement.
Maybe "GSoC Project Intro. External Workspace Manager Plugin"